### PR TITLE
[OS-913, OS-912] - Add support for OpenCV along with a small Webcam sample

### PR DESCRIPTION
**Pull request by skarbat**

### CODE REVIEW

File: debian/changelog [hunk 1]
```diff
@@ -1,3 +1,9 @@
+kano-jupyter (1.0-1) unstable; urgency=low
```
Comment (tombettany) [inline debian/changelog]: Doesn't need a new version as we've not ever released this, just merge this with the previous entry. Also, the version should match the next OS version to release so change this file to just be 

```
kano-jupyter (4.3.0-0) unstable; urgency=low

  * Added initial PoC integration on Kano OS
  * Added Python2 Kernel and a sample Python Opencv notebook

 -- Team Kano <dev@kano.me>  Thu, 14 Mar 2019 16:09:00 +0100
```

File: debian/control [hunk 2]
```diff
@@ -7,5 +7,17 @@ Build-Depends: debhelper (>=9.0.0)
...
+   jupyter-notebook,
+   jupyter-nbextension-jupyter-js-widgets,
+   python-ipykernel,
+   python-opencv,
+   python-numpy,
+   python-nltk,
+   python3-nltk,
+   python-bs4,
+   python3-bs4,
+   python-matplotlib
```
Comment (tombettany) [inline debian/control]: Nice, much cleaner like this.

**Review by tombettany — APPROVED** [review]
Just a minor `debian/changelog` problem, otherwise looks good. Great stuff getting this in so quickly.

File: debian/changelog (hunk 1 above)
Comment (skarbat) [reply debian/changelog]: Good point, rectified

File: debian/control (hunk 2 above)
Comment (skarbat) [reply debian/control]: Thanks!

**Pull request merged**